### PR TITLE
edit

### PR DESCRIPTION
**Pull request by Alzubair98**
I'm trying to move all the files from the feature branch to the main branch . 
everything is okay and everything works perfectly fine.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+text.md
```
Comment (Munsa1) [inline .gitignore]: - [x] After you install linters on your local machine, a node_modules folder is generated. That file should also be added to the .gitignore file. Kindly add it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,97 @@
-# Hello---micro
-First project with microverse
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
```
Comment (Munsa1) [inline README.md]: - [x] The template for the readme file to be used is as follows, [follow this link](https://github.com/microverseinc/readme-template/blob/master/README.md).
- [x] Copy the template by clicking the two boxes in the image below once you go to the link above.
image

- [x] Then clear your readme file and paste the template in your readme file on your macjine.
- [x] Lastly edit the readme file by adding project name, git hub link and linkedin link, as well as the issues link for your current project.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,97 @@
...
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
```
Comment (Munsa1) [inline README.md]: - [x] Optional
I will put this requirement here but it has nothing to do with this part. Kindly add meaningful messages when making commits. An example is `git commit -m "Add HTML file and CSS file"`, that way anyone going through your work will have an idea of where a particular change was introduced to the code just by checking your commit message
Your commit says `edit`, which doesn't necessarily explain anything
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Alzubair98 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark:  Passing linters
:heavy_check_mark:  Correct Gitflow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Naming convention for branch names should be meaningful to the task you are performing. for example for this project, you could have named the branch as `hello-setup`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Alzubair98) [reply .gitignore]: hi thanks i have done most of the requirements, but where can i find the node-modules, im really sorry for this question but this is my first time using the linters @Munsa1

File: README.md (hunk 3 above)
Comment (Alzubair98) [reply README.md]: dear i have done exactly what you asked me to do 
thanks @Munsa1

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Alzubair98 👋

Your project is complete! ✔️ 🏁. This is the First Project 1️⃣ You get Approved ✔️! in microverse 🌐.

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ no linter errors
- ✔️ descriptive commit messages
- ✔️ good communication in PR

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**